SPECS-EXTENDED/mbedtls/mbedtls.spec

```
@@ -0,0 +1,301 @@
+Summary:        Light-weight cryptographic and SSL/TLS library
+Name:           mbedtls
+Version:        2.28.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.trustedfirmware.org/projects/mbed-tls
+Source:         https://github.com/Mbed-TLS/%{name}/archive/refs/tags/v%{version}.tar.gz#%{name}-%{version}.tar.gz
```

I had missed the `/` in my previous review
```suggestion
Source:         https://github.com/Mbed-TLS/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

---

Updated.